logic/placements.py

```diff
@@ -64,23 +77,23 @@ def norm_values(norm: Callable[[str], EIN]):
     "Beedle's Shop - 50 Rupee Item",
     "Beedle's Shop - 1000 Rupee Item",
 ]
-VANILLA_BEEDLE_PLACEMENT = norm_vanilla(BEEDLE_CHECKS)
+VANILLA_BEEDLE_PLACEMENT = norm_restrict_vanilla(BEEDLE_CHECKS)
```

Comment:
```suggestion
VANILLA_BEEDLE_PLACEMENT = norm_force_vanilla(BEEDLE_CHECKS)
```
This can be problematic: Currently, beedle isn't patched at all, when vanilla placement is selected (and it also makes pouch necessary to buy the pouch upgrades). Even if a different item than the vanilla one is randomized in beedle's shop, it wouldn't appear in game

Reply:
Now the comment saying `norm_force_vanilla` is unused can be removed